sass/MagicBox/MagicBox.scss

```diff
@@ -120,6 +120,14 @@
     }
     .magic-box-suggestions.magic-box-hasSuggestion {
       display: block;
+      .coveo-magicbox-suggestions {
+        border: 1px solid $grey-500;
```

Comment:
Are you sure this border color fits well with the rest of the interface borders? can you provide a quick screenshot? 🙏

Reply:
Without the querySuggestPreview
![Capture d’écran, le 2019-08-01 à 16 45 06](https://user-images.githubusercontent.com/50176606/62326327-d17b3300-b47b-11e9-86e2-49bf64bacec9.png)

With the querySuggestPreview
![Capture d’écran, le 2019-08-01 à 16 45 26](https://user-images.githubusercontent.com/50176606/62326361-e788f380-b47b-11e9-9473-49b6d380baa7.png)